common/cmake/devices.cmake

```diff
@@ -1519,8 +1684,14 @@ function(ADD_FPGA_TARGET)
     # Generate FASM
     # -------------------------------------------------------------------------
     set(OUT_FASM ${OUT_LOCAL}/${TOP}.fasm)
+    set(OUT_FASM_MERGED ${OUT_LOCAL}/${TOP}.merged.fasm)
     set(OUT_FASM_CONCATENATED ${OUT_LOCAL}/${TOP}.concat.fasm)
     set(OUT_FASM_GENFASM ${OUT_LOCAL}/${TOP}.genfasm.fasm)
+    set_target_properties(${NAME} PROPERTIES
+      OUT_FASM ${OUT_FASM}
```

Comment:
If you want to do this, see "add_output_to_fpga_target", and use that.  This will not work as you expect.

Reply:
You need to remove all of these properties